assets/airports/ebbr.json

```diff
@@ -1,241 +1,1872 @@
         },
         {
-            "name": ["25L", "07R"],
-            "name_offset": [[0, 0], [0, 0]],
-            "end": [["N50d53m56.19", "E004d31m23.88", "159ft"], ["N50d53m21.89", "E004d28m55.40", "175ft"]],
-            "delay": [40, 20],
-            "ils": [true, false]
+            "name": ["07L", "25R"],
+            "end": [
+                ["N50d54m00.12", "E004d27m34.43", "141ft"],
+                ["N50d54m41.55", "E004d29m57.79", "141ft"]
+            ],
+            "ils": [false, true]
         },
         {
-            "name": ["25R","07L"],
-            "name_offset": [[0, 0], [0, 0]],
-            "end": [["N50d54m41.55", "E004d29m57.79", "110ft"], ["N50d54m00.12", "E004d27m34.43", "129ft"]],
-            "delay": [10, 30],
-            "ils": [true, false]
+            "name": ["07R", "25L"],
+            "end": [
+                ["N50d53m21.89", "E004d28m55.40", "141ft"],
+                ["N50d53m56.19", "E004d31m23.88", "141ft"]
+            ],
+            "ils": [false, true]
         }
     ],
     "sids": {
-        "NORTH": {
-            "icao": "NORTH",
-            "name": "NORTH",
-            "rwy": {
-                "19" : ["NORTH"],
-                "01" : ["NORTH"],
-                "25L": ["NORTH"],
-                "25R": ["NORTH"],
-                "07L": ["NORTH"],
-                "07R": ["NORTH"]
-            },
-            "draw": [["EBBR","NORTH"]]
-        },
-        "EAST": {
-            "icao" : "EAST",
-            "name" : "EAST",
-            "rwy": {
-                "19" : ["EAST"],
-                "01" : ["EAST"],
-                "25L": ["EAST"],
-                "25R": ["EAST"],
-                "07L": ["EAST"],
-                "07R": ["EAST"]
-            },
-            "draw": [["EBBR","EAST"]]
-        },
-        "SOUTH": {
-            "icao": "SOUTH",
-            "name": "SOUTH",
-            "rwy": {
-                "19" : ["SOUTH"],
-                "01" : ["SOUTH"],
-                "25L": ["SOUTH"],
-                "25R": ["SOUTH"],
-                "07L": ["SOUTH"],
-                "07R": ["SOUTH"]
-            },
-            "draw": [["EBBR","SOUTH"]]
-        },
-        "WEST": {
-            "icao": "WEST",
-            "name": "WEST",
-            "rwy": {
-                "19" : ["WEST"],
-                "01" : ["WEST"],
-                "25L": ["WEST"],
-                "25R": ["WEST"],
-                "07L": ["WEST"],
-                "07R": ["WEST"]
-            },
-            "draw": [["EBBR","WEST"]]
+        "LNO": {
+            "icao": "LNO",
+            "name": "Olno",
+            "suffix": {"01":"6F", "07L":"5H", "07R":"5Y", "19":"6L", "25L":"5Q", "25R":"5C"},
+            "rwy": {
+                "01": ["HUL06", ["R045", "A60"]],
+                "07L": ["AFI17", "AFI22", "REMBA"],
+                "07R": ["BUB04", "BUB08", "REMBA"],
+                "19": ["BR010", ["BR011", "A60"]],
+                "25L": [],
+                "25R": []
```

Comment:
It's valid for this runway, yet there's only one waypoint generated if they use it. So that will have to be addressed somehow. If it's a case where a heading is used, we are not far away from being able to support that.